### ISSUE TITLE
Any possibility of collaboration with FreeCAD_AirPlaneDesign WB?

### ISSUE THREAD

**Issue by luzpaz** [chbergmann/CurvedShapesWorkbench#1]
https://github.com/FredsFactory/FreeCAD_AirPlaneDesign

CC @FredsFactory

**Comment by chbergmann** [comment]
yes, the AirPlaneDesign workbench could use wing profile sketches to connect the foils instead of connecting by straight lines. Unfortunately, the workbench is too french for me.
